althea_kernel_interface/src/open_tunnel.rs

```diff
@@ -63,6 +63,8 @@ pub struct TunnelOpenArgs<'a> {
     pub private_key_path: &'a Path,
     /// our mesh ipv6 address
     pub own_ip: IpAddr,
+    /// our second mesh ip used for exit roaming
```

Comment:
this should not that it's only used on the exits as a transition method and needs to be removed after all routers are updated to beta 20. We'll need a vector on the root ip parameter in the client exit config so that we can expand maintain reliability if one exit is down or unreachable.